app/views/browse/_pod-details.html

```diff
@@ -79,7 +79,7 @@
       <div ng-if="pod.spec.volumes.length">
```

Comment:
not new, but this makes the below link to attach storage unreachable, also missing the canI check on that link